submissions/7SergSmith7/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>HTML-CSS-Popup</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav class="right_navigation">
+          <a href="#" class="right_navigation__mail">Mail</a>
+          <a href="#" class="right_navigation__images">Images</a>
+          <input type="checkbox" id="popup-check" class="popup_btn" />
```

Comment:
Navigation is best used in this format, the sample from [MDN](https://developer.mozilla.org/uk/docs/Web/HTML/%D0%95%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82/nav)
```
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```